docs/001_develop/03_client-capabilities/008_interaction/slider.mdx

```diff
@@ -0,0 +1,178 @@
+      }
+    }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+    Declaration
+    ```html
+    <rapid-slider>
+      <rapid-slider-label position="0"> Low </rapid-slider-label>
+      <rapid-slider-label position="50"> Mid </rapid-slider-label>
+      <rapid-slider-label position="100"> High </rapid-slider-label>
+    </rapid-slider>
+    ```
+
+    Usage
+    ```typescript
+    export function MyComponent() {
+      const handleChange = (event) => {
+        console.log({ event.target.value })
+      };
+      return (
+        <rapid-slider onChange={handleChange}>
+          <rapid-slider-label position="0"> Low </rapid-slider-label>
+          <rapid-slider-label position="50"> Mid </rapid-slider-label>
+          <rapid-slider-label position="100"> High </rapid-slider-label>
+        </rapid-slider>
+      );
+    }
+  ```
+</TabItem>
+<TabItem value="angular">
+
+  Declaration
+  ```html
+  <rapid-slider>
+    <rapid-slider-label position="0"> Low </rapid-slider-label>
+    <rapid-slider-label position="50"> Mid </rapid-slider-label>
+    <rapid-slider-label position="100"> High </rapid-slider-label>
+  </rapid-slider>
+  ```
+
+  Usage
+  ```typescript
+  import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+  import { FormsModule } from '@angular/forms';
+  @Component({
+    selector: 'my-root',
+    template: `
+              <rapid-slider (change)="handleChange($event)">
+                <rapid-slider-label position="0"> Low </rapid-slider-label>
+                <rapid-slider-label position="50"> Mid </rapid-slider-label>
+                <rapid-slider-label position="100"> High </rapid-slider-label>
+              </rapid-slider>
+              `,
+    standalone: true,
+    schemas: [CUSTOM_ELEMENTS_SCHEMA],
+    imports: [FormsModule],
+  })
+  export class AppComponent {
+    handleChange(event: Event) {
+      const target = event.target as HTMLInputElement;
+      console.log(target.value);
+    }
+  }
+  ```
+</TabItem>
+</Tabs>
+
+Use of the `<rapid-slider-label>` is optional.
+
+## API
+
+Property and attribute binding examples for Genesis Component syntax.
+
+| Name        | Type      | Description                                                   | Examples |
+|-------------|-----------|---------------------------------------------------------------|---|
+| disabled    | `boolean` | Similar to `readonly`, but with a blur layer in the component. | `<rapid-slider disabled>` |
+| form        | `string`  | Associates this component to a form. Form `id` needs to be passed. If no Id informed, then it will be associated with the ancestor form. | |
+| max         | `number`  | Defines maximum number of the slider.                          | `<rapid-slider max=50>` |
+| min         | `number`  | Defines minimum number of the slider.                          | `<rapid-slider min=10>` |
+| step        | `number`  | Defines the value of each step of the slider.                  | `<rapid-slider step=5>` |
```

Comment:
Can we get a number of examples with the step values, min max, vertical/hoirzontal etc